neuro_san/service/agent_service.py

```diff
@@ -102,17 +94,18 @@ def get_request_count(self) -> int:
         """
         return self.request_counter.get_count()
 
-    # pylint: disable=no-member
-    def Function(self, request: service_messages.FunctionRequest,
-                 context: grpc.ServicerContext) \
-            -> service_messages.FunctionResponse:
+    def function(self, request_dict: Dict[str, Any],
+                 request_metadata: Dict[str, Any],
+                 context: Any) \
+            -> Dict[str, Any]:
         """
```

Comment:
For all our API methods: we switch to Dict-based parameters and results.
And rename method names to make pylons happy.

Reply:
"context" here is still gRPC artifact, but only because I don't want to go down leaf-server-common code
and fix things there. This is for later.